### ISSUE TITLE
Support for editable state of a field

### ISSUE THREAD

**Issue by saberraz** [MerginMaps/mobile#2607]
With the introduction of https://github.com/qgis/QGIS/pull/51525, it would be good to have support for this feature in the app, so the users can set the state of field edit based on expression.

**Comment by TomPohys** [comment]
+1 for this enhancement! Any plan to implement this?

Thank you

**Comment by tomasMizera** [comment]
Yeah, a nice one indeed! I'll add it to dev queue for Q4 / Q1 '25

**Comment by tomasMizera** [comment]
Hi @TomPohys - this one is now released in 2025.1.0!

**Comment by TomPohys** [comment]
Awesome! Thanks!
